integrationtesting/fixtures/src/main/java/org/opendcs/fixtures/configuration/Configuration.java

```diff
@@ -1,4 +1,4 @@
-package org.opendcs.spi.configuration;
+package org.opendcs.fixtures.configuration;
```

Comment:
I don't see an issue with moving this interface, but I've been keeping "spi" in the package name intentionally as a hint to the reader.

Does make sense to keep under fixtures but perhaps "...fixtures.spi..."?